src/main/resources/locales/de/plugins.json

```diff
@@ -24,6 +24,10 @@
       "close": "Schließen",
       "error": "Das Senden der Test E-Mail ist fehlgeschlagen.",
       "success": "Die E-Mail wurde erfolgreich versandt."
+    },
+    "topics": {
+      "header": "Abonnierte Kanäle",
+      "helpText": "Hier können Sie auswählen, zu welchen Themen Sie Benachrichtigungen per Mail erhalten wollen."
```

Comment:
Maybe use passive wording? Not sure though.

Reply:
For example `Zu den ausgewählten Themen werden Benachrichtigungen per Mail verschickt`?

Reply:
I like this better.